utils/index.py

```diff
@@ -0,0 +1,97 @@
+"""
+    Python script to find broken links from a csv file and try match the filenames
+    to existing backup files
+
+    Example Usage: python index.py --csv input.csv --out output.csv
+
+    Outputs a csv with meeting_id, url, filesystem_paths, possible_duplicate columns
+"""
+import argparse
+import csv
+import glob
+import hashlib
+import os
+from collections import namedtuple
+
+
+def get_backup_file_path(path):
+    with open(path, 'r') as f:
+        reader = csv.reader(f)
+        for row in reader:
+            yield row[0], row[2]
```

Comment:
https://docs.python.org/3/library/csv.html#csv.DictReader is your friend - next time - no need to change now